### PR TITLE
Change "Top X" to "Up to Top X" In Events Tab #7853

### PR DESCRIPTION
**Pull request by Carter-Thomas**

### CODE REVIEW

**Comment by coder13** [comment]
Can you provide a screenshot? It'd be nice if screenshots can be provided for FE tickets.

**Comment by coder13** [comment]
I think we can also benefit from having some explanation text explaining the caveat for the advancement

image

E.g. something like "a minimum of 25% of participants must be eliminated going into the next round."

@gregorbg thoughts?

**Comment by Carter-Thomas** [comment]
Sure!

Proposed: 
Screenshot (447)

Current:
Screenshot (448)

File: WcaOnRails/app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -221,7 +221,8 @@ def do_add
     registration_row[:wca_id]&.upcase!
     registration_row[:email]&.downcase!
     person_details = {
-      name: registration_row[:name],
+      first_name, last_name = registration_row[:name].split(' ', 2),
```
Comment (coder13) [inline WcaOnRails/app/controllers/registrations_controller.rb:224]: Is this supposed to be in this PR?

File: WcaOnRails/config/locales/en.yml
Comment (coder13) [inline WcaOnRails/config/locales/en.yml:1]: Is this supposed to be in this PR?

File: WcaOnRails/app/controllers/registrations_controller.rb (hunk 1 above)
Comment (Carter-Thomas) [reply WcaOnRails/app/controllers/registrations_controller.rb:224]: No, it's getting randomly added for whatever reason

Comment (gregorbg) [reply WcaOnRails/app/controllers/registrations_controller.rb:224]: Then just rebase on master and make sure that it's not included in this PR :)

File: WcaOnRails/config/locales/en.yml
Comment (gregorbg) [reply WcaOnRails/config/locales/en.yml:1]: I agree that the changes to the mail subjects should be part of a separate PR

**Review by gregorbg — CHANGES_REQUESTED** [review]
Thank you for your contribution! Please clean up the unrelated changes in the git diff

**Comment by gregorbg** [comment]
Was it your intention to close this PR?

**Pull request closed without merging**